src/app/fillLeaderboard.js

```diff
@@ -0,0 +1,44 @@
+        <li class='leaderboardItem whoseThatPokemonLeaderboard'><img src='./static/assets/ui/hall-of-fame/bronze.png' /><span class='leaderboardPlayerName'>${pokemonApiRanking.mode1.scores[2] ? pokemonApiRanking.mode1.scores[2].name : "-"}</span><span class='leaderboardPoints'>${pokemonApiRanking.mode1.scores[2] ? pokemonApiRanking.mode1.scores[2].score : "-"}</span></li>`
+
+    // add what it looks like mode leaderboard
+    leaderboardTable.innerHTML += 
+        `<li class='leaderboardItem whatItLooksLikeLeaderboard leaderFirstPlace'><img src='./static/assets/ui/hall-of-fame/gold.png' /><span class='leaderboardPlayerName'>${pokemonApiRanking.mode2.scores[0] ? pokemonApiRanking.mode2.scores[0].name : "-"}</span><span class='leaderboardPoints'>${pokemonApiRanking.mode2.scores[0] ? pokemonApiRanking.mode2.scores[0].score : "-"}</span></li>
+        <li class='leaderboardItem whatItLooksLikeLeaderboard'><img src='./static/assets/ui/hall-of-fame/silver.png' /><span class='leaderboardPlayerName'>${pokemonApiRanking.mode2.scores[1] ? pokemonApiRanking.mode2.scores[1].name : "-"}</span><span class='leaderboardPoints'>${pokemonApiRanking.mode2.scores[1] ? pokemonApiRanking.mode2.scores[1].score : "-"}</span></li>
+        <li class='leaderboardItem whatItLooksLikeLeaderboard'><img src='./static/assets/ui/hall-of-fame/bronze.png' /><span class='leaderboardPlayerName'>${pokemonApiRanking.mode2.scores[2] ? pokemonApiRanking.mode2.scores[2].name : "-"}</span><span class='leaderboardPoints'>${pokemonApiRanking.mode2.scores[2] ? pokemonApiRanking.mode2.scores[2].score : "-"}</span></li>`
+
+    // add guess the type mode leaderboard
+    leaderboardTable.innerHTML += 
+        `<li class='leaderboardItem guessTheTypeLeaderboard leaderFirstPlace'><img src='./static/assets/ui/hall-of-fame/gold.png' /><span class='leaderboardPlayerName'>${pokemonApiRanking.mode3.scores[0] ? pokemonApiRanking.mode2.scores[0].name : "-"}</span><span class='leaderboardPoints'>${pokemonApiRanking.mode3.scores[0] ? pokemonApiRanking.mode2.scores[0].score : "-"}</span></li>
+        <li class='leaderboardItem guessTheTypeLeaderboard'><img src='./static/assets/ui/hall-of-fame/silver.png' /><span class='leaderboardPlayerName'>${pokemonApiRanking.mode3.scores[1] ? pokemonApiRanking.mode2.scores[1].name : "-"}</span><span class='leaderboardPoints'>${pokemonApiRanking.mode3.scores[1] ? pokemonApiRanking.mode2.scores[1].score : "-"}</span></li>
+        <li class='leaderboardItem guessTheTypeLeaderboard'><img src='./static/assets/ui/hall-of-fame/bronze.png' /><span class='leaderboardPlayerName'>${pokemonApiRanking.mode3.scores[2] ? pokemonApiRanking.mode2.scores[2].name : "-"}</span><span class='leaderboardPoints'>${pokemonApiRanking.mode3.scores[2] ? pokemonApiRanking.mode2.scores[2].score : "-"}</span></li>`
```

Comment:
Looks like a big repetition of the **nearly** the same code. Besides classes and mode-accessors.

Can you try to create some function (functions?) that will take care of that and eliminate repetitions?

Example use case: somebody wants to remove css `leaderboardPlayerName`. 
Right now he must change 9 places. Can you rewrite it in a way that the person need to change:
(easier version) only 3 places?
(harder version) only 1 place?

Reply:
also, adding some formatting to this html would be helpful :) it is hard to read nested HTML that is in one line